src/app/TelemetryApp.svelte

```diff
@@ -24,29 +24,27 @@ function updateQueryString() {
   }
 }
 
-$: if (visible) {
+$: if ($store.token && visible) {
```

Comment:
is it necessary to check `$store.token` here?

Reply:
I think so. If `$store.token` is removed, `updateQueryString` does run when the user is unauthenticated. It doesn't seem to cause any harm, but it does seem unnecessary. Open to suggestions. :smiley:

Reply:
will `$store.token` ever be removed?

Reply:
Shouldn't be. It's set if the user authenticates and unset otherwise.